spec/models/event_spec.rb

```diff
@@ -22,4 +22,10 @@
     event.published = true
     expect(event).to be_valid
   end
+
+  it "checks bust_event" do
+    perform_enqueued_jobs do
```

Comment:
This spec seems to only check that the callbacks run without raising an exception implicitly, but doesn't actually check that the cache was busted as it states in the description. If you want to check that the cache busting you can check that the job was enqueued when `event.save` is called. I think there's no need to actually run the jobs in the test (with `perform_enqueued_jobs`) here.

Reply:
I did that. And checked all method names are ok in the various job specs.